bin/bootstrax

```diff
@@ -1095,14 +1122,27 @@ def process_run(rd, send_heartbeats=args.production):
         if not osp.exists(dd['location']):
             fail(f"No access to live data folder {dd['location']}")
 
-        thread_info = rd.get('daq_config', dict()).get('processing_threads', dict())
-        n_readout_threads = sum([v for v in thread_info.values()])
         try:
-            daq_chunk_duration = int(rd['daq_config']['strax_chunk_length'] * 1e9)
-            daq_overlap_chunk_duration = int(rd['daq_config']['strax_chunk_overlap'] * 1e9)
+            # Fetch parameters from the rundoc. If not readably, let's use redax' default
+            # values (that are hardcoded here).
+            if 'daq_conig' not in rd:
+                log_warning('No daq_config in the rundoc!')
+            dq_conf = rd.get('daq_config', dict())
+            to_read = ('processing_threads', 'strax_chunk_length','strax_chunk_overlap',
+                       'strax_fragment_payload_bytes')
+            for conf in to_read:
+                if conf not in dq_conf:
+                    log_warning(f'{conf} not in rundoc for {run_id}! Using default values.')
```

Comment:
The default value for `processing_threads` is complicated.  The default is 8 threads per instance of redax, but finding _that_ value is rather more involved.

Reply:
at the moment we would get None or 0 using this logic if it's not specified

Reply:
I'm just wondering if there's a cleaner way to handle this field, because using the "default" value is not as straightforward as for the other fields.